testdata/generate/all.pb.go

```diff
@@ -1,4 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.25.0
```

Comment:
this will probably fail the next time `protoc-gen-go` is updated, but that's ok?